src/main/java/nl/ordina/jobcrawler/controller/scraper/HuxleyITResponse.java

```diff
@@ -0,0 +1,54 @@
+package nl.ordina.jobcrawler.controller.scraper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import nl.ordina.jobcrawler.model.Vacancy;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class HuxleyITResponse {
+
+    private Integer hits;
+    private ArrayList<Map<String, Object>> vacanciesData;
+
+    /*
+        This annotation indicates to Jackson that something must be executed upon the "result" parameter of the returned JSON.
+
+        This method extracts the needed vacancy data from the "result" data and stores it as List of key-value pairs to return.
+
+        An interesting point is that Jackson converts the JSON objects to Map<String, Object>, where the keys are the parameter names.
+
+        The JSON that Jackson has parsed has this format (only the parameters used by us are noted here):
+        {
+            "result": {
+                "hits": 60,
+                "results": [
+                    {
+                        "jobReference": "HA-XXXXXXXX_XXXXXXXXXX",
+                        "title": "XXXXXX",
+                        "city": "XXXXXX",
+                        "salaryText": "XXXXXX",
+                        "postDate": "2020-04-14T07:41:38.0679209Z",
+                        "description": "XXXXXX"
+                    }
+                ]
+            }
+        }
+     */
```

Comment:
Good documentation. Perhaps it's best to write it in official Javadoc notation?

Reply:
Changed to javadoc notation!